src/drivers/net/virtio/mod.rs

```diff
@@ -65,8 +65,13 @@ impl RxQueues {
 		//
 		let packet_size = if dev_cfg.features.contains(virtio::net::F::MRG_RXBUF) {
 			1514
-		} else {
+		} else if dev_cfg.features.contains(virtio::net::F::GUEST_TSO4)
+			|| dev_cfg.features.contains(virtio::net::F::GUEST_TSO6)
+			|| dev_cfg.features.contains(virtio::net::F::GUEST_UFO)
+		{
 			dev_cfg.raw.as_ptr().mtu().read().to_ne().into()
```

Comment:
If MTU is 65550, should we not add the size of `virtio_net_hdr` (12 Bytes) here?